activesupport/lib/active_support/message_encryptor.rb

```diff
@@ -28,6 +29,16 @@ def self.dump(value)
       end
     end
 
+    module AeadVerifier #:nodoc:
```

Comment:
This is used in tandem with AEAD ciphers, but it isn't an AEAD verifier—it's a no-op verifier.